src/components/WalletAuthModal.tsx

```diff
@@ -101,6 +98,7 @@ export const WalletAuthModal = ({ open }: { open: boolean }) => {
     }
 
     try {
+      setConnectMessage('');
```

Comment:
@levity this line is still needed, because the WalletAuthModal closes and then this tries to update the text on it (unmounted now) and throws a console error. I think its easiest to just bring this over to your PR rather than keep this one open since you moved all this code.

Reply:
pushed these fixes onto your branch: https://github.com/coordinape/coordinape/pull/1690/commits/6b781f07c101b32708e3bb0665907900538804ed, so I'll go ahead and clsoe this PR. @levity